docs/docs/APIdocs/layers/nn/Loss Functions/CosineEmbeddingCriterion.md

```diff
@@ -0,0 +1,77 @@
+## CosineEmbeddingCriterion ##
+
+**Scala:**
+```scala
+val cosineEmbeddingCriterion = CosineEmbeddingCriterion(val margin: Double = 0.0, val sizeAverage: Boolean = true)
```

Comment:
If a user copy/paste this code to create an instance, he will get errors. use below instead. 
```scala
val cosineEmbeddingCriterion = CosineEmbeddingCriterion(margin = 0.0, sizeAverage = true)
```